modules/express/src/clientRoutes.ts

```diff
@@ -453,6 +453,20 @@ async function handleV2ConsolidateUnspents(req: express.Request) {
   return wallet.consolidateUnspents(req.body);
 }
 
+/**
+ * handle wallet fanout unspents
+ * @param req
+ */
+async function handleV2ConsolidateAccount(req: express.Request) {
+  const bitgo = req.bitgo;
+  const coin = bitgo.coin(req.params.coin);
+  if (coin.getFamily() !== 'hbar') {
```

Comment:
Please use `CoinFamily.HBAR` from `@bitgo/statics` - these are subject to change.